src/Neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -103,10 +107,18 @@ public sealed class PolicyContract : NativeContract
         /// </summary>
         private const string MillisecondsPerBlockChangedEventName = "MillisecondsPerBlockChanged";
 
+        private const string WhitelistChangedEventName = "WhitelistChanged";
```

Comment:
We should rename this event. Because we may have other "whitelist" in the future.